package.json

```diff
@@ -41,9 +41,9 @@
   },
   "devDependencies": {
     "@types/jest": "^29.2.1",
-    "@types/node": "^17.0.5",
+    "@types/node": "20.0.0",
     "@typescript-eslint/parser": "^5.8.0",
-    "@vercel/ncc": "^0.33.1",
+    "@vercel/ncc": "^0.38.1",
     "axios-mock-adapter": "^1.21.2",
     "eslint": "^8.26.0",
```

Comment:
Have you considered updating the versions of other modules? For instance, the current version for [@types/jest](https://www.npmjs.com/package/@types/jest) is 29.5.12, aligning with the same major version. This alignment suggests that there shouldn't be any breaking changes.

When transitioning to a more recent version of Node.js, I suggest checking for updates in other modules as well, provided it doesn't introduce issues or conflicts with the project. This approach ensures that we benefit from new fixes and address any potential vulnerabilities.

I think you can run a `npm update` or `yarn update` to update the modules ([docs](https://docs.npmjs.com/cli/v8/commands/npm-update)). Also, you can check if there are any vulnerabilities to fix by running `npm audit` ([docs](https://docs.npmjs.com/cli/v9/commands/npm-audit)).

Reply:
After trying to update other modules running `npm update`, it looks like the `eslint` package started to fail, so I will keep the other versions as they are at the moment.
<img width="1159" alt="Screenshot 2024-02-02 at 14 01 37" src="https://github.com/vmware-labs/vmware-image-builder-action/assets/91888629/a0570fb7-9415-4e1a-ad68-0aa67cb0e3ba">

Reply:
mmm... the message says "@typescript-eslint/eslint-plugin" couldn't be found, so it should be fixed by adding that module to `package.json`. I do not see the reason for not adding it.